.travis.yml

```diff
@@ -8,6 +8,8 @@ notifications:
   email: false
 before_script:
 - npm prune
+- link-parent-bin -s true
+- link-parent-bin -s true -c babel
```

Comment:
demos and website also must be linked for publishing purposes. and btw why not using npm run link here?